packages/flame/lib/src/gestures/events.dart

```diff
@@ -0,0 +1,183 @@
+import 'package:flutter/gestures.dart';
+
+import '../../extensions.dart';
+
+abstract class _BaseInfo<T> {
```

Comment:
I would rather this not be private, that would allow people to do generic handlers

Reply:
This inheritance was done most because of DRY. I am not sure if generics handles would have any use here.

Reply:
Changed me mind, made this public.